.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Describe your changes
+
+<!-- Please add a link to a ticket if relevant: eg: "closes #340" -->
+
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code
+- [ ] If it is a core feature, I have added tests.
+- [ ] Do we need to implement analytics?
```

Comment:
What does this mean?

Reply:
I donno, it came from a template 🤷 

Mostly, I think of it as—is there tracking that needs to be implemented alongside the business logic?

Reply:
I think it's maybe overkill - otherwise ship it?

Reply:
I think it's a bit open to interpretation so maybe not super helpful